File: Assignment_33/Program3.cpp
/*
Write a program which accepts one number from user and check whether 9th or 12th bit is ON or OFF.
Input :257
Output : true
*/

#include<iostream>
using namespace std;

typedef unsigned int UINT;

bool ChkBit(UINT iNo)
{
    UINT iMask1 = 256;
    UINT iMask2 = 2048;
    UINT iMask = 0;
    UINT iResult = 0;

    iMask = iMask1 | iMask2;
    iResult = iNo & iMask;

    return iResult;
}

int main()
{
    UINT iValue = 0;
    bool bRet = false;

    cout<<"Enter the Number:";
    cin>>iValue;

    bRet = ChkBit(iValue);

    if(bRet == true)
    {
        cout<<"Either 9th or 12th Bit is ON";
    }
    else
    {
        cout<<"Both 9th and 12th Bit is OFF";
    }

    return 0;
}

File: Assignment_48/Program3.java
/* Write a java program which accepts string from user and display below pattern.
Input :  Hello
Output : H
         H  e  
         H  e  l
         H  e  l  l
         H  e  l  l  o
*/

import java.util.*;

class Pattern
{
    public void Pattern(String str)
    {
        int i = 0,j = 0;

        for(i = 0;i < str.length();i++)
        {
            for(j = 0;j <= i;j++)
            {
                System.out.print(str.charAt(j)+"\t");
            }
            System.out.println();
        }
    }
}

class Program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the String : ");
        String str = sobj.nextLine();

        Pattern pobj = new Pattern();
        pobj.Pattern(str);
    }
}